### ISSUE TITLE
Live Feed

### ISSUE THREAD

**Issue by Scaprod** [godarklight/DarkMultiPlayer#389]
Live feed is not existent, I was playing with a friend and it was boring because... In the Atmosphere, there was no live feed. Will this be fixed?

**Comment by RockyTV** [comment]
What do you mean with live feed?

**Comment by furyanPDX** [comment]
I think he's referring to the ability to watch your friends launch craft or fly planes. Maybe a spectator mode of some kind?

**Comment by RockyTV** [comment]
What do you mean with live feed? Do you mean in atmosphere spawning?

**Comment by 81ninja** [comment]
@furyanPDX There IS a spectator mode but vessels will only be synced to the server and between players after they leave the "safety bubble" around the KSC runway or launch pad.
So you can't see the launch from the beginning at the launch pad or runway but once your friend leaves KSC you can see his/her craft, in atmosphere or otherwise.

**Comment by furyanPDX** [comment]
Right, I get that :) I was just clarifying for @RockyTV how I interpreted the OP in question.

**Comment by Scaprod** [comment]
It is how you can interact with other players' crafts, not really a spectator mode, you can dock atmospherically with Live Feed and even more, my friend and I were having a hard time playing together because of this live feed issue.

**Comment by 81ninja** [comment]
@Scaprod I for one don't really understand what you mean by "live feed". You want to have a camera showing your friend's point of view, like a picture-in-picture image? Or something like that?

Or you just mean your friend's ship seems to jump around while you are near it, instead of flying smoothly alongside your own ship?
